app/assets/javascripts/govuk_publishing_components/lib/govspeak/magna-charta.js

```diff
@@ -379,7 +407,18 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
   }
 
   MagnaCharta.prototype.insert = function () {
-    this.$table.insertAdjacentElement('afterend', this.$graph)
+    var label = document.createElement('span')
+    var labelId = 'mc-chart-not-accessible-' + this.chartId
+
+    label.innerHTML = 'This content is not accessible - switch to table'
+    label.className = 'mc-hidden'
```

Comment:
Hey @owenatgov testing on VoiceOver works really well, all makes sense liking the button switch over. 👏🏼

I'm noticing with JAWS this particular `span` copy is not being read out so I can't hear 
`This content is not accessible - switch to table`  

I think it's because this is using `display: none` instead of something like ` govuk-visually-hidden`

Reply:
Ahaaaaaa that's a good point. I'll amend and test 👍

Reply:
This is a bit tricky it turns out. If I replace the `mc-hidden` on the label for the graph container with `govuk-visually-hidden` then screen readers (tested with VO) read out that label as a separate element, so the announcements go:

1. "This content is not accessible - please switch to table, group"
2. *down one* "This content is not accessible - please switch to table"
3. *down one* "Out of group, This content is not accessible - switch to table"

I've played around with this, hiding the visually hidden label inside an `aria-hidden` but I can't find a sweet spot. I personally feel like out of these probs, JAWS not reading that this content isn't accessible and VO reading the label again, The former seems like the least damaging. What do you reckon? Should I go back to the drawing board? Have I understood the issue properly?

Reply:
Further to our conversation balancing all the variables of this element, the impact of this issue highlighted plus zooming out and thinking about magna chart + govspeak in general this is in a really good place 🚀